README.md

```diff
@@ -89,3 +89,4 @@ $ search video property
 | 0.0.6   | Add Pybites Podcast search                  |
 | 0.0.7   | Add Pybites tips search                     |
 | 0.0.8   | Move podcast feed parsing to our platform   |
+| 0.0.9   | Add tox + testing to support 3.9 + 3.10     |
```

Comment:
maybe worth to create a changelog at some point (e.g. at `0.1`)

Reply:
yes, there must be a better way to do this :)

Reply:
opened issue